src/zimdump.cpp

```diff
@@ -39,13 +41,38 @@
 # include <windows.h>
 #else
 # define SEPARATOR "/"
-# include <unistd.h>
-# include <sys/stat.h>
+#include <unistd.h>
 #endif
 
+#define ERRORSDIR "_exceptions/"
```

Comment:
We have to agree with the doc in #89 
But I don't know what is ok (the code or the doc)

Reply:
The code is OK, I ask for this name `_exceptions`.